Mathlib/Order/Hom/Basic.lean

```diff
@@ -1220,3 +1220,11 @@ lemma denselyOrdered_iff_of_strictAnti {X Y F : Type*} [LinearOrder X] [Preorder
     rw [hf.le_iff_ge]
 
 end DenselyOrdered
+
+universe v u in
+
```

Comment:
```suggestion
```
I think that if you use the `in` keyword, there should be no line break before the declaration, just like we do for `variable`.